src/image.tsx

```diff
@@ -62,9 +63,11 @@ const ImageError = ({noBorderRadius}: ImageErrorProps) => {
                 display: 'flex',
                 alignItems: 'center',
                 justifyContent: 'center',
-                backgroundColor: isInverse
-                    ? vars.colors.backgroundSkeletonInverse
-                    : vars.colors.backgroundSkeleton,
+                backgroundColor: !isDarkMode
```

Comment:
hmmm... why do we need to check `isDarkMode` here @aweell? shouldn't the colors just work in dark mode?

Reply:
In movistar, for dark mode, `backgroundSkeletonInverse` color is `movistarBlueDark`. We don't want that to be shown in the background of the card.

Reply:
Some values weren't working as expected. The team has been reviewing them and here's a fix to avoid the `darkMode` check:

https://github.com/Telefonica/mistica-design/pull/1236/files

Reply:
Fixed

Reply:
nice, thank you @aweell